src/odemis/driver/test/spectrometer_test.py

```diff
@@ -252,6 +255,27 @@ def test_vbinning(self):
         md = data.metadata
         self.assertEqual(md[model.MD_BINNING], tuple(binning))
 
+        # reduce the binning to just the CCD max binning
```

Comment:
It would be nice to have a bit more info what it tested for, as it is hard to follow that way. The binning of the spectrometer?